Base/QTApp/qSlicerMainWindow.cxx

```diff
@@ -461,7 +461,7 @@ void qSlicerMainWindowPrivate::setupUi(QMainWindow * mainWindow)
     this->PythonConsoleToggleViewAction->setText(qSlicerMainWindow::tr("&Python Interactor"));
     this->PythonConsoleToggleViewAction->setToolTip(qSlicerMainWindow::tr(
       "Show Python Interactor window for controlling the application's data, user interface, and internals"));
-    this->PythonConsoleToggleViewAction->setShortcut(QKeySequence("Ctrl+3"));
+    this->PythonConsoleToggleViewAction->setShortcuts({qSlicerMainWindow::tr("Ctrl+3"), qSlicerMainWindow::tr("Ctrl+`")});
```

Comment:
Also add this to the user manual (user_interface.md).

Reply:
Done. The user interface page now shows:
![image](https://user-images.githubusercontent.com/15837524/154321730-fd49fb3e-bf14-4c74-af9b-a6a460eb4ad1.png)